src/node/gov/handlers/proposals.h

```diff
@@ -215,7 +215,11 @@ namespace ccf::gov::endpoints
             v.set(
               "member_id",
               js_context.new_string_len(member_id.data(), member_id.size()));
-            v.set_bool("vote", vote_result);
+            if (v.set_bool("vote", vote_result) != 1)
```

Comment:
Why does this instance warrant a check, but none of the other `set` calls do? Looks like they all return a similar result code.